### ISSUE TITLE
Discoverability: tooltips in Scope

### ISSUE THREAD

**Issue by iamtesch** [HebiRobotics/scope2-feedback#77]
Issues that came up from customer feedback today

Adding mouseover text to a number of the options in Scope would make the tool more discoverable.  Especially for concepts like "persist", or "command lifetime", which are not perfectly clear from their names, this would help a user play around with Scope with a little more guidance.

**Comment by ennerf** [comment]
Where does Scope use the concept of "command lifetime"? Scope always sends zero lifetime and there is no way to change it.

I agree with persist. What other things do you think need a mouseover?

**Comment by iamtesch** [comment]
As many places as possible, honestly.

As for the "command lifetime", sorry I was unclear -- I was referring to the state feedback for command lifetime/lockout.  Right now, it just has this under "Locked":
![lockes_state](https://user-images.githubusercontent.com/16693168/59611799-014bc280-90ea-11e9-8767-d70a13200796.png)

But for a list of some places where I think tooltips could be good from a quick glance _just at the left pane_:

- what are the groupings?  What do they mean?  This might not be necessary.
- What does "Device Search" mean?  What is this actually doing?  Not at all intuitive for new user since nothing seems to change when you toggle it.
- "Clear" is confusing to a new user, because it may just cause the link above to blink.  Probably need to reference what a stale module is, etc.

Basically, on pretty much every panel I think there are a number of places where we could add really helpful content.  Just try to look from the lens of a new user.  Persist is obviously a big one, but I think ~25% of the buttons/status text we have could be really confusing.  I think tooltips are an unobtrusive way to add more info (they don't change the basic GUI at all), and I wouldn't mind having them on everything.
